### ISSUE TITLE
Leo items "in" Earley sets

### ISSUE THREAD

**Issue by dabrahams** [jeffreykegler/Marpa-arxiv-paper#6]
In §6:

> In each Earley set, there is at most one Leo item per symbol.

An Earley set is a set of Earley items, so IIUC, I might rephrase this as,  

> Associated with each Earley set, there is a mapping M from symbol to at most one Leo item.

In §6.1, 

> `predecessor`LIMT ∈ `component-orig`ES

uses the same “Leo item in earley set“ idea, which is a bit confusing.

**Comment by jeffreykegler** [comment]
Fixed with commits https://github.com/jeffreykegler/Marpa-arxiv-paper/commit/6e2b4f7dbcbf0c0137e58d61bdfc79c3ce3ee4b1 and https://github.com/jeffreykegler/Marpa-arxiv-paper/commit/6f75d408261fdd2f8dc72be0ea91cd74eddf612b.  Absent feedback to the contrary, I will close this issue after a couple of days.

**Comment by jeffreykegler** [comment]
Closed per https://github.com/jeffreykegler/Marpa-arxiv-paper/issues/6#issuecomment-1235765750.
